packages/loader/container-loader/src/container.ts

```diff
@@ -480,7 +481,6 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 {
                     eventName: "ContainerClose",
                     sequenceNumber: error.sequenceNumber ?? this._deltaManager.lastSequenceNumber,
-                    loading: !this.loaded,
```

Comment:
Is this always false or something? Why remove it?

Reply:
Now this same payload is part of every event, so no need to have it here